### PR TITLE
Use newly introduced "initialize a sensor object" operation

### PR DESCRIPTION
**Pull request by anssiko**
PTAL @riju @alexshalamov @pozdnyakov

(For the context, see https://github.com/w3c/sensors/pull/345)

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/magnetometer/pull/40.html" title="Last updated on Feb 26, 2018, 4:39 PM GMT (474792a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/magnetometer/40/cbb1c32...474792a.html" title="Last updated on Feb 26, 2018, 4:39 PM GMT (474792a)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -313,16 +327,24 @@ Abstract Operations {#abstract-opertaions}
 <div algorithm="construct a magnetometer object">
 
     : input
+    :: |magnetometer_interface|, a {{Magnetometer}} [=interface=] [=identifier=] or
+       an {{UncalibratedMagnetometer}} [=interface=] [=identifier=].
     :: |options|, a {{MagnetometerSensorOptions}} object.
     : output
     :: A {{Sensor}} object.
```
Comment (pozdnyakov) [inline index.bs]: a {{Magnetometer}} object?

Comment (alexshalamov) [reply index.bs]: I think Sensor object is correct as an output, or use something like:

`:: Instance of an |magnetometer_interface|`

Comment (anssiko) [reply index.bs]: Output can be either `Magnetometer` or `UncalibratedMagnetometer`, both of which are also instances of a `Sensor`. Not sure how to best spec this, to be explicit maybe:
```
:: A {{Magnetometer}} or {{UncalibratedMagnetometer}} object.
```

Comment (anssiko) [reply index.bs]: Fix https://github.com/w3c/magnetometer/pull/40/commits/474792adcb75b961267abd573bb41574b3a6b541

**Pull request merged**